.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: release
+
+on:
+  push:
+    tags:
+      - v*.*.*
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checout Code
+        uses: actions/checkout@v2
+      - name: Set up JDK ${{ matrix.java-version }}
```

Comment:
Where is `java-version` defined?

Reply:
In the `matrix`. If you look at the `ci.yml` workflow you will see it there as well.

Reply:
I doubt those are actually shared between workflows. Have you verified this?

Reply:
I haven't verified that, but inside the ci workflow (https://github.com/dnsimple/dnsimple-java/blob/main/.github/workflows/ci.yml) there is no setup of the matrix either, so my guess is that it's somewhere else and the workflows feed from it

Reply:
> I haven't verified that, but inside the ci workflow (https://github.com/dnsimple/dnsimple-java/blob/main/.github/workflows/ci.yml) there is no setup of the matrix either, so my guess is that it's somewhere else and the workflows feed from it

It's right here:

https://github.com/dnsimple/dnsimple-java/blob/main/.github/workflows/ci.yml#L34-L37

Reply:
Working while having a your son visiting from London is proving to have a dumbing effect... 😂 

Sorted!

Reply:
Thanks. What's the point of using a Matrix so different version numbers) to publish a given package? I suggest to just take _a_ version instead of using a matrix.

The other clients probably have the same _problem_.